src/components/__tests__/Alert.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { shallow } from "enzyme";
```

Comment:
Update this test spec to the one in pagination widget

Reply:
Updated

Reply:
Updated